src/android/CustomFCMReceiverPlugin.java

```diff
@@ -107,29 +55,22 @@ private boolean inspectAndHandleMessageData(Map<String, String> data) throws JSO
         if (type.equals("incoming_phone_call") || type.equals("incoming_video_call")) {
             isHandled = true;
 
-            Bundle extras = new Bundle();
-            extras.putString("from", payload.getString("from"));
-            extras.putString("payload", payloadString);
-
-            tm.addNewIncomingCall(handle, extras);
-            tm.showInCallScreen(false);
-            openFacetalkApp();
+            Intent intent = new Intent("INCOMING_CALL_INVITE");
+            intent.setComponent(new ComponentName(this.applicationContext, MyConnectionService.class));
+            intent.putExtra("payload", payloadString);
+            Log.d(TAG, "launching startService() intent for MyConnectionService...");
+
+            // When you call startService() for an Android Service that is already running, a new instance of the service is not created.
+            // Instead, the onStartCommand() method of the existing service instance is called again.
+            // This allows you to deliver a new Intent to the running service,
+            // enabling it to process new requests or update its state without creating redundant instances.
+            // The ConnectionService needs to be started if for any reason its not currently running.
+            this.applicationContext.startService(intent);
```

Comment:
Can resolve this if it's not a problem/solved, but please verify that when the app is closed/phone is locked that the MyConnectionService is able to spin up and handle the incoming intent.

I vaguely remember having issues because the Firebasex service that processes the notification doesn't have CordovaCall running, which is why I added the `openFacetalkApp()` there.